resources/services/ec2/key_pairs.go

```diff
@@ -0,0 +1,78 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Ec2KeyPairs() *schema.Table {
```

Comment:
Kindly add `cq-gen` config for this resource.
Additionally, the generate instruction comment should be present here, and no diff should occur once you run this generate command on this file.

See [AWS SES Template resource](https://github.com/cloudquery/cq-provider-aws/blob/main/resources/services/ses/templates.go#L13) for inspiraition.

Reply:
I will add an entry to the `resources/services/ec2/gen.hcl` file for this resource. I notice that there are other resources with missing entries in this file. Can you provide me more info about the purpose of this file?

Reply:
We tend to use `cq-gen` to generate resources (may be installed via `make install-tools`).
Additionally, it provides an easy way to specify the resource structure.

Also, we are going to check for drift changes for the generated resources (so you should be able to run `go generate` on this file and see no changes whatsoever).